Antidote/CallControlsView.h

```diff
@@ -15,11 +15,31 @@
 - (void)callControlsMicButtonPressed:(CallControlsView *)callsControlView;
 - (void)callControlsSpeakerButtonPressed:(CallControlsView *)callsControlView;
 - (void)callControlsResumeButtonPressed:(CallControlsView *)callsControlView;
+- (void)callControlsVideoButtonPressed:(CallControlsView *)callsControlView;
+- (void)callControlsEndCallButtonPressed:(CallControlsView *)callsControlView;
 
 @end
 
+typedef NS_ENUM(NSUInteger, CallsControlsViewType) {
+    /**
+     * Use this view for audio controls.
+     */
+    CallsControlsViewTypeExpand,
+
+    /**
+     * Use this view when showing video.
+     * Controls are set up horizontally |-video-mic-speaker-endCall-|
```

Comment:
I think we want to use `compact` mode for horizontal audio as well. Comment is wrong then.